modules/unsupported/wfs-ng/src/main/java/org/geotools/data/wfs/internal/v2_0/Capabilities200ServiceInfo.java

```diff
@@ -62,9 +68,18 @@ public String getDescription() {
         if (serviceIdentification == null) {
             return null;
         }
-        @SuppressWarnings("unchecked")
-        List<String> abs = serviceIdentification.getAbstract();
-        return abs == null || abs.isEmpty() ? null : abs.get(0);
+
+        // The Abstract is of Type LanguageStringType, not String.
+        StringBuilder sb = new StringBuilder();
+        for (Object line : serviceIdentification.getAbstract()) {
+            if (line instanceof LanguageStringType) {
+                sb.append(((LanguageStringType) line).getValue());
+            } else {
```

Comment:
is line likely to be something other than a LanguageStringType?

Reply:
Variable line is of type Object. So it is safer programming to allow it to be something different than LanguageStringType, too.